misk/src/main/kotlin/misk/prometheus/PrometheusHistogramRegistry.kt

```diff
@@ -10,12 +10,10 @@ class PrometheusHistogramRegistry : HistogramRegistry {
   override fun newHistogram(
     name: String,
     help: String,
-    labelNames: List<String>,
-    buckets: DoubleArray?
+    labelNames: List<String>
   ): PrometheusHistogram {
     val builder =
-        io.prometheus.client.Histogram.build(name, help).labelNames(*labelNames.toTypedArray())
-    if (buckets != null) builder.buckets(*buckets)
+        io.prometheus.client.Summary.build(name, help).labelNames(*labelNames.toTypedArray())
```

Comment:
The user-defined quantiles need to be added here. The Prometheus client library does not have default quantiles, so without defining any you only get sum and count.